src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -106,12 +106,17 @@ internal partial class BoundFixedLocalCollectionInitializer
 
     internal partial class BoundStackAllocArrayCreation
     {
-        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Count);
+        internal static ImmutableArray<BoundExpression> GetChildInitializers(BoundArrayInitialization arrayInitializer)
+        {
+            return arrayInitializer?.Initializers ?? ImmutableArray<BoundExpression>.Empty;
+        }
+
+        protected override ImmutableArray<BoundNode> Children => StaticCast<BoundNode>.From(GetChildInitializers(this.InitializerOpt).Insert(0, this.Count));
```

Comment:
Are the `Children` properties covered by test?

Reply:
Not directly, but the elements appears in IOperation tests.